async/amqp/amqp.go

```diff
@@ -18,6 +18,11 @@ import (
 	"github.com/streadway/amqp"
 )
 
+const (
+	// AMQPConsumerComponent definition.
+	AMQPConsumerComponent = "amqp-consumer"
```

Comment:
if this is not used outside of the package we can unexport it and remove the AMQP prefix since it already lives in the amqp package

Reply:
Regarding this and I agree with removing the prefix to avoid stuttering. 

On the other hand, I think we should keep it exported. This way people using these packages will have a uniform way available to refer to the component names (eg. while logging) which will correspond to the name used in tracing.

Reply:
@mantzas any thoughts?

Reply:
this is actually internal and should probably not be used outside the package. We can always export it if somebody wants it for some reason.